builtin/credential/ldap/path_config.go

```diff
@@ -0,0 +1,143 @@
+package ldap
+
+import (
+	"net/url"
+	"strings"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfig(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: `config`,
+		Fields: map[string]*framework.FieldSchema{
+			"url": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "ldap URL to connect to (default: ldap://127.0.0.1)",
+			},
+			"domain": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "LDAP domain to use (eg: dc=example,dc=org)",
+			},
+			"userattr": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Attribute used for users (default: cn)",
+			},
+		},
```

Comment:
@rasky It looks like this is not yet well handled across the auth providers. Don't worry about it for now, we will do a sweep through them to tighten this up.